services/src/main/java/org/jboss/windup/web/services/rest/RegisteredApplicationEndpointImpl.java

```diff
@@ -20,9 +20,12 @@
     private RegisteredApplicationService registeredApplicationService;
 
     @Override
-    public Collection<RegisteredApplication> getAllApplications()
+    public Collection<RegisteredApplication> getAllApplications(Long projectId)
     {
-        return this.registeredApplicationService.getAllApplications();
+        if (projectId == null)
```

Comment:
Path of least resistance. If I changed other paths, I would have to go through all components and search where the path is used. Since we don't use a single constant and build them by concatenation, this seemed easier.